kitchensink/src/app/common/PageElements/FoundationElements/ColorBox/ColorBox.jsx

```diff
@@ -5,47 +5,50 @@ import classNames from 'classnames';
 import './colorBox.css';
 
 const ColorBox = (props) => (
-  <div styleName="colorBoxContainer">
-    {props.title && <div styleName="title">{props.title}</div>}
+  <div styleName={classNames('colorBoxContainer', { [`${props.size}`]: true })} style={props.style ?? props.style}>
```

Comment:
you can just do like this (without the template litteral :

`{ [props.size]: true }`

Reply:
✅